doc/api/n-api.md

```diff
@@ -189,6 +215,26 @@ typedef void (*napi_async_complete_callback)(napi_env env,
                                              void* data);
 ```
 
+#### napi_threadsafe_function_call_js
+Function pointer used with asynchronous thread-safe function calls. The callback
+will be called on the main thread. Its purpose is to call the JavaScript
+function. N-API sets up the environment prior to calling this callback, so it is
+sufficient to call `napi_call_function` with `js_callback` after having computed
+the parameters the JavaScript function is to have.
+Callback functions must satisfy the following signature:
+```C
+typedef void (*napi_threadsafe_function_call_js)(napi_env env,
+                                                 napi_value js_callback,
+                                                 void* context,
+                                                 void* data);
+```
+- `[in] env`: The environment to use for API calls.
+- `[in] js_callback`: The JavaScript function to call.
+- `[in] context`: The optional data with which the thread-safe function was
+created.
+- `[in] data`: Data created by the secondary thread, to be converted to
```

Comment:
How about

Data created by the secondary thread.  It is the responsibility of the callback to convert this native data to JavaScript values (with napi functions) that can be passed as parameters when js_callback is invoked.